apps/qubit/modules/informationobject/actions/uploadFindingAidAction.class.php

```diff
@@ -49,7 +49,15 @@ public function execute($request)
 
       if ($this->form->isValid() && isset($file))
       {
-        $tempPath = $file->getTempName();
+        $i18n = $this->context->i18n;
+
+        $params = array(
+          'objectId'    => $this->resource->id,
+          'description' => $i18n->__('Uploading finding aid for: ').$this->resource->getTitle(array('cultureFallback' => true)),
```

Comment:
I think this is more correct.

```
'description' => $i18n->__('Uploading finding aid for: %1%', array('%1%' => $this->resource->getTitle(array('cultureFallback' => true)))),
```

It enables the translator to arrange the sentence in the way it's needed.